helpers/networks.ts

```diff
@@ -101,6 +101,24 @@ export const NETWORKS: {
     explorerURL: "https://blockscout.moonriver.moonbeam.network",
     rpcUrls: ["https://eth-iris.taikai.network:8080"]
   },
+  "0x507": {
+    id: "1287",
+    name: "Moonbase Testnet",   
+    key: "moonbase",
+    currency: ETH,
+    decimals: 18,
+    explorerURL: "https://blockscout.moonriver.moonbeam.network",
+    rpcUrls: ["https://rpc.api.moonbase.moonbeam.network"]
+  },
+  "0x504": {
+    id: "1284",
+    name: "Moonbeam Testnet",   
+    key: "moonbeam",
+    currency: ETH,
+    decimals: 18,
+    explorerURL: "https://blockscout.moonriver.moonbeam.network",
+    rpcUrls: ["https://rpc.api.moonbase.moonbeam.network"]
```

Comment:
the RPC is wrong

Reply:
what is the correct?

![image](https://user-images.githubusercontent.com/34746557/193321384-bea15a7c-274f-4544-b848-09456d3bc2db.png)